InstantPrintPlugin.py

```diff
@@ -15,7 +15,6 @@
 import os
 
 from InstantPrintTool import InstantPrintTool
-import resources_rc
```

Comment:
Why do you remove this line?

Reply:
I removed it because it was unused according to pep8.